### DISCUSSION TITLE
RecordPlugin does not have pauseRecording() is there an alternative?

### DISCUSSION THREAD

**Discussion by AmitJoki** [katspaugh/wavesurfer.js#3247] [Q&A]
The use case is to be able to pause/resume recording before the user decides to stop the recording.

**Comment by katspaugh** [comment]
I don’t know of an alternative. Not even sure it’s possible. The media recorder data comes in chunks and I don’t know if you can skip chunks without compromising the integrity of the file.

**Reply by AmitJoki** [reply]
The media recorder already has a native pause method https://developer.mozilla.org/en-US/docs/Web/API/MediaRecorder/pause Can we not use it?

**Reply by katspaugh** [reply]
Ah, good. Then feel free to add it to the plugin, I'd be happy to review the PR.
